test/conf/logback-test.xml

```diff
@@ -38,7 +38,7 @@
     </triggeringPolicy>
 
     <encoder>
-      <pattern>%-5level [%thread] %date{ISO8601} %msg%n</pattern>
+      <pattern>%-5level [%thread] %date{ISO8601} %F:%L - %msg%n</pattern>
```

Comment:
what does it do?

Reply:
it makes it on par with console output.
Specifically, it adds file name and line of the log statement emission